app/containers/HomeContainer/index.js

```diff
@@ -91,13 +106,23 @@ export function HomeContainer({
                 <T id="matching_repos" values={{ totalCount }} />
               </div>
             )}
-            {items.map((item, index) => (
+            {/* {items.map((item, index) => (
```

Comment:
delete this. we shouldn't have any commented-out unused code in the repo

Reply:
i resolve that.